sdk/src/metrics/export/periodic_exporting_metric_reader.cc

```diff
@@ -167,10 +167,36 @@ bool PeriodicExportingMetricReader::OnForceFlush(std::chrono::microseconds timeo
   }
 
   is_force_flush_notified_.store(false, std::memory_order_release);
+
+  if (timeout == std::chrono::steady_clock::duration::zero())
```

Comment:
Do we need remove this if branch?It does nothing.

Reply:
yes removed thanks.